src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
```

Comment:
According to the checklist, you should not begin class or method implementation with an empty line. Please remove the empty lines before the class declaration.